nixos/modules/services/networking/mosquitto.nix

```
@@ -233,15 +232,50 @@ in
         ExecStart = "${pkgs.mosquitto}/bin/mosquitto -c ${mosquittoConf}";
         ExecReload = "${pkgs.coreutils}/bin/kill -HUP $MAINPID";
 
-        ProtectSystem = "strict";
-        ProtectHome = true;
+        # Hardening
+        CapabilityBoundingSet = "";
```

Should this be
```suggestion
        CapabilityBoundingSet = [ ];
```
so that merging works?

---

Hell no. https://github.com/NixOS/nixpkgs/issues/120617